test/topology_custom/test_mv_delete_partitions.py

```diff
@@ -0,0 +1,59 @@
+
+from test.pylib.manager_client import ManagerClient
+
+import asyncio
+import pytest
+import time
+import logging
+
+logger = logging.getLogger(__name__)
+
+async def insert_with_concurrency(cql, key, values, concurrency):
+    logger.info(f"Starting writes for key {key} with concurrency {concurrency}")
+    async def do_inserts(i: int):
+        # Insert all rows with keys with the same modulo as i
+        while i < values:
+            await cql.run_async(f"INSERT INTO ks.tab (key, c, v) VALUES ({key}, {i}, {key + i})")
```

Comment:
It took too long in the first version, now it takes 10 min in debug mode on my machine and considerably less in dev/release. Will see how fast will it be on jenkins - it might need a longer timeout for debug